pkg/build/util/jenkins.go

```diff
@@ -0,0 +1,134 @@
+package util
+
+import (
+	"fmt"
+
+	"github.com/golang/glog"
+	"github.com/openshift/origin/pkg/api/latest"
+	"github.com/openshift/origin/pkg/client"
+	serverapi "github.com/openshift/origin/pkg/cmd/server/api"
+	"github.com/openshift/origin/pkg/config/cmd"
+	"github.com/openshift/origin/pkg/template"
```

Comment:
@mfojtik This is causing some test failures:

```
# github.com/openshift/origin/pkg/template
import cycle not allowed in test
package github.com/openshift/origin/pkg/template (test)
	imports github.com/openshift/origin/pkg/api/install
	imports github.com/openshift/origin/pkg/build/api/install
	imports github.com/openshift/origin/pkg/build/api/v1
	imports github.com/openshift/origin/pkg/build/util
	imports github.com/openshift/origin/pkg/template
```

Reproducible with `make WHAT=pkg/template test-unit`